### ISSUE TITLE
Error: posting as3 config failed for tenants:(Sample_01) with error: HTTP 422 :: {"code":422,"message":"Invalid request value '[object Object]' (path: /targetTokens/controls) : should be string {\"type\":\"string\"}"}

### ISSUE THREAD

**Issue by papineni87** [F5Networks/terraform-provider-bigip#511]
### Environment

 * Terraform bigip provider Version:  1.9.0

### Summary

Unable to post AS3 declaration  from bigip provider when AS3 json has target tokens

{
   "class": "AS3",
   "action": "deploy",
   "targetUsername": "xxxxxx",
   "trace": true,
   "persist": true,
   "targetTokens": {
       "X-F5-Auth-Token": "xxxxxxx"
   },
   "declaration": {
      "class": "ADC",
      "schemaVersion": "3.22.1",
      "id": "urn:uuid:33045210-3ab8-4636-9b2a-c98d22ab915d",
      "label": "Sample 1",
      "remark": "Simple HTTP application with RR pool",
      "Sample_01": {
         "class": "Tenant",
         "A1": {
            "class": "Application",
            "my-virtual": {
               "class": "Service_HTTP",
               "virtualAddresses": [
                  "x.x.x.x"
               ],
               "pool": "web_pool"
            },
            "web_pool": {
               "class": "Pool",
               "monitors": [
                  "http"
               ],
               "members": [{
                  "servicePort": 80,
                  "serverAddresses": [
                     "x.x.x.x",
                     "x.x.x.x"
                  ]
               }]
            }
         }
      }
   }
}
 
This results in the following error message:
 
Error: posting as3 config failed for tenants:(Sample_01) with error: HTTP 422 :: {"code":422,"message":"Invalid request value '[object Object]' (path: /targetTokens/controls) : should be string {\"type\":\"string\"}"}

### Steps To Reproduce

1.  Any As3 json has target tokens in the declaration

### Expected Behavior
As3 declaration should be configured successfully on bigip

### Actual Behavior

This results in the following error message:
 
Error: posting as3 config failed for tenants:(Sample_01) with error: HTTP 422 :: {"code":422,"message":"Invalid request value '[object Object]' (path: /targetTokens/controls) : should be string {\"type\":\"string\"}"}

**Comment by papineni87** [comment]
Tracking with Infraano-401

**Comment by codecowboydotio** [comment]
I too have hit this bug.

It appears as if the controls portion is being added to the declaration multiple times.

Using the declaration:
```
{
   "class": "AS3",
   "action": "deploy",
   "targetUsername": "admin",
   "trace": true,
   "persist": true,
   "targetTokens": {
       "X-F5-Auth-Token": "ZBVWAQYU3FRRTFXOTR5DOUJ2TG"
   },
   "declaration": {
      "class": "ADC",
      "schemaVersion": "3.22.1",
      "id": "urn:uuid:33045210-3ab8-4636-9b2a-c98d22ab915d",
      "label": "Sample 1",
      "remark": "Simple HTTP application with RR pool",
      "Sample_01": {
         "class": "Tenant",
         "A1": {
            "class": "Application",
            "my-virtual": {
               "class": "Service_HTTP",
               "virtualAddresses": [
                  "10.0.1.10"
               ],
               "pool": "web_pool"
            },
            "web_pool": {
               "class": "Pool",
               "monitors": [
                  "http"
               ],
               "members": [{
                  "servicePort": 80,
                  "serverAddresses": [
                     "192.0.1.10",
                     "192.0.1.11"
                  ]
               }]
            }
         }
      }
   }
}

```

I can see in the terraform trace output that the controls header is added incorrectly.
It appears to have been added inside the target tokens component.
```
2021-06-18T19:36:32.254+1000 [DEBUG] provider.terraform-provider-bigip_v1.9.0: 2021/06/18 19:36:32 [INFO] Creating as3 config in bigip:{"action":"deploy","class":"AS3","declaration":{"Sample_01":{"A1":{"class":"Application","my-virtual":{"class":"Service_HTTP","pool":"web_pool","virtualAddresses":["10.0.1.10"]},"web_pool":{"class":"Pool","members":[{"serverAddresses":["192.0.1.10","192.0.1.11"],"servicePort":80}],"monitors":["http"]}},"class":"Tenant"},"class":"ADC","controls":{"class":"Controls","userAgent":"Terraform/1.0.0/terraform-provider-bigip/1.9.0"},"id":"urn:uuid:33045210-3ab8-4636-9b2a-c98d22ab915d","label":"Sample 1","remark":"Simple HTTP application with RR pool","schemaVersion":"3.22.0"},"persist":true,"targetTokens":{"X-F5-Auth-Token":"ZBVWAQYU3FRRTFXOTR5DOUJ2TG","controls":{"class":"Controls","userAgent":"Terraform/1.0.0/terraform-provider-bigip/1.9.0"}},"targetUsername":"admin","trace":true}
```

Likewise when I explicitly set the controls variable, I get the following:
```
2021-06-18T19:48:06.249+1000 [DEBUG] provider.terraform-provider-bigip_v1.9.0: 2021/06/18 19:48:06 [INFO] Creating as3 config in bigip:{"action":"deploy","class":"AS3","controls":{"class":"Controls","controls":{"class":"Controls","userAgent":"Terraform/1.0.0/terraform-provider-bigip/1.9.0"},"userAgent":"svk"},"declaration":{"Sample_01":{"A1":{"class":"Application","my-virtual":{"class":"Service_HTTP","pool":"web_pool","virtualAddresses":["10.0.1.10"]},"web_pool":{"class":"Pool","members":[{"serverAddresses":["192.0.1.10","192.0.1.11"],"servicePort":80}],"monitors":["http"]}},"class":"Tenant"},"class":"ADC","controls":{"class":"Controls","userAgent":"Terraform/1.0.0/terraform-provider-bigip/1.9.0"},"id":"urn:uuid:33045210-3ab8-4636-9b2a-c98d22ab915d","label":"Sample 1","remark":"Simple HTTP application with RR pool","schemaVersion":"3.22.1"},"persist":true,"targetTokens":{"X-F5-Auth-Token":"ZBVWAQYU3FRRTFXOTR5DOUJ2TG","controls":{"class":"Controls","userAgent":"Terraform/1.0.0/terraform-provider-bigip/1.9.0"}},"targetUsername":"admin","trace":true}
```
You can see above that the control class is added multiple times.

I have also tested the AS3 declaration from curl and postman with a valid token and it works without issue - which leads me to believe this is a provider problem.

**Comment by RavinderReddyF5** [comment]
fixed in v1.10.0, please check and open new issue if u see any problem.
